resources/camino-trekker/components/Map/Map.vue

```diff
@@ -126,7 +126,7 @@ onMounted(() => {
 provide(MapInjectionKey, mapRef);
 </script>
 
-<style scoped>
+<style>
```

Comment:
Curious about this change - is this tied to the vite change or unrelated?

Reply:
Oh just saw the commit text. Seems sketchy though - would be good to understand the boundaries of this issue if we’re moving forward with vite.

Reply:
Here's what I think was going on:

When building for production, the mapbox container wouldn't be visible, causing tests to fail.

Looking at the CSS in dev tools, instead of getting the `70vh` (set in another component), it got this `100%` or overwrote the 70vh, causing the map to not appear.

I think the scoped styles caused the same specificity to the mapbox container, and then production build compiled this CSS after the 70vh CSS, overwriting it.

Where to place the blame is an open question – it could be vite loading thing in different order, or it could be me not being clear about my intended specificity. 

There were several ways to resolve specificity: increase it in each parent component, lower it here, set a 70vh here as well... 

Considering the mapbox container can be used elsewhere, I thought the scoping was overkill and just removed it, lowering the specificity.